src/pyOpenMS/tests/unittests/test_spectrum_experiment.py

```
@@ -0,0 +1,1084 @@
+    assert isinstance(aligner(rich_spec, rich_spec), float)
+
+@report
+def testChromatogramToosl():
+    """
+    @tests: ChromatogramTools
+     ChromatogramTools.__init__
+     ChromatogramTools.convertChromatogramsToSpectra
+     ChromatogramTools.convertSpectraToChromatograms
+    """
+    pyopenms.ChromatogramTools()
+    pyopenms.ChromatogramTools.convertChromatogramsToSpectra
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)